src/frequenz/sdk/timeseries/_ringbuffer/buffer.py

```diff
@@ -568,11 +568,12 @@ def __getitem__(self, index_or_slice: SupportsIndex | slice) -> float | FloatArr
         """
         return self._buffer.__getitem__(index_or_slice)
 
-    def __len__(self) -> int:
-        """Return the amount of items that this container currently holds.
+    @property
+    def valid_size(self) -> int:
```

Comment:
Same comments about property and the name, feel free to resolve this comment so we can keep the discussion in one place.